components/prism-core.js

```diff
@@ -549,5 +548,5 @@ if (typeof module !== 'undefined' && module.exports) {
 
 // hack for components to work correctly in node.js
 if (typeof global !== 'undefined') {
-	global.Prism = Prism;
+	global['Prism'] = Prism;
```

Comment:
Not a fan of whatever's causing this. I saw this below as well.

Reply:
That's a dirty hack to remove an error message from [TSCheck](https://www.typescriptlang.org/docs/handbook/type-checking-javascript-files.html).

Not a fan either btw.